### PR TITLE
ConnectionList done

### PR DESCRIPTION
**Pull request by emir4ik-code**

### CODE REVIEW

**Review by raylu511 — DISMISSED** [review]

**Comment by Matt-Tan15** [comment]
can you provide a screenshot?

**Comment by emir4ik-code** [comment]
nah

**Comment by emir4ik-code** [comment]
Screenshot 2023-01-06 161750

**Review by raylu511 — APPROVED** [review]

**Comment by emir4ik-code** [comment]
New screenshot
Screenshot 2023-01-06 173059

**Comment by ZohaibManzoor00** [comment]
Looks good. Nit: the message icon can be moved down a bit

**Review by ZohaibManzoor00 — APPROVED** [review]

File: src/shared/Mocks/Mocks.ts [hunk 1]
```diff
@@ -1,41 +1,62 @@
-const userOneProfilePic =
+const profilePic =
   "https://scontent-lga3-1.xx.fbcdn.net/v/t39.30808-6/255700933_3056178267990272_2842286584796059820_n.jpg?_nc_cat=100&ccb=1-7&_nc_sid=09cbfe&_nc_ohc=A80f0zuNGfAAX9PJM9Q&_nc_ht=scontent-lga3-1.xx&oh=00_AfDKwIGJBkGU3eDCRmQDF8-_TV57x4I8XmqSOgR_EadBUg&oe=63BD1968";
 
 export const messageArray = [
```
Comment (Matt-Tan15) [inline src/shared/Mocks/Mocks.ts:4]: what is the message array for?

Comment (emir4ik-code) [reply src/shared/Mocks/Mocks.ts:4]: testing

**Review by Matt-Tan15 — APPROVED** [review]
looks good

**Pull request merged**